### ISSUE TITLE
AssetCollector compression not working anymore

### ISSUE THREAD

**Issue by Patta** [a-r-m-i-n/min#19]
TYPO3 11.5.27
PHP 8.1.18
min 2.1.2
composer installers v4

Since v2.1.2 the compression of assetcollector assets (css and js) does not work anymore. The assets are just ignored.
It seems that commit https://github.com/a-r-m-i-n/min/commit/2c92ec0e58bf2b503759d70c468c289dc4d82e46 introduced a regression.

With v2.1.1
```
<link media="all" href="/typo3temp/assets/compressed/femanager-min.css.1685534495.gzip" rel="stylesheet">
<script src="/typo3temp/assets/compressed/Femanager.min-min.js.1685534495.gzip"></script>
```

With v2.1.2
```
<link media="all" href="/_assets/c1c23d805658c7c8c9054ca1396e79a3/Css/Extensions/femanager.1685107140.css" rel="stylesheet">
<script src="/_assets/f038457a108ded7ca0b077b63bfae195/JavaScript/Femanager.min.1682508541.js"></script>
```

Downgraded to v2.1.1 as temp workaround.

**Comment by a-r-m-i-n** [comment]
Thanks for reporting. We fixed the asset collector compression with release 3.0 for TYPO3 12.

**Comment by Patta** [comment]
Thanks for your feedback. It would be great if the bugfix could also be provided for TYPO3 v11. For those who can't upgrade to v12 yet. Thanks.

**Comment by a-r-m-i-n** [comment]
I've backported the bugfix and released it with version 2.1.3

**Comment by Patta** [comment]
thanks a lot!
